### DISCUSSION TITLE
Users changing their passwords

### DISCUSSION THREAD

**Discussion by alanzaf** [Libki/libki-server#342] [Q&A]
Hi,

does anybody know if is there a way for users to change their passwords? I haven't found it, and i think it would be quite convenient.

Thanks in advance for your attention

**Comment by kylemhall** [comment]
There isn't! It's a bit tricky with Libki. We don't want to allow password changes for users verified against SIP or other external services, but I don't see why we can't get password changing on the user side implemented! Can you file an issue for that?

**Reply by alanzaf** [reply]
Sure, i will. Thanks for your attention 😃

**Comment by sekjal** [comment]
This is now being tracked as an Issue, so closing the discussion here
